cli/src/main/java/dev/starfix/Starfix.java

```diff
@@ -211,11 +218,17 @@ public static void editConfig() throws Exception {
             // user--------------
             while (true) {
                 System.out.println("\n--------Enter preferred Clonning path--------");
-                clone_path = reader.readLine();
+                String clonePathInput = reader.readLine();
+                if(clonePathInput == null || clonePathInput.isEmpty()){
+                    System.out.println("No input provided for Clone Path:Now using Existing/Default Config");
+                    break;
+                }
                 // We'll check if the path enterd by user is a valid path or not
-                File tmp = new File(clone_path);
-                if (tmp.exists())
+                File tmp = new File(clonePathInput);
```

Comment:
instead of requiring the location to exist we should just create it.

Reply:
Sure i was thinking that too. I was wondering about the possibility when the user inputs some invalid path like `/MyProject` . If he enters `~/` before this path it will be valid I guess but not without . In invalid cases I'll ask user to renter the path.

Reply:
Done!

Reply:
oops!! don't know why the Github Actions build failed with compilation error. Its works on my local . Strange !!